spec.md

```diff
@@ -377,24 +377,31 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
+
+It is useful for the Platform for the Broker to define exactly what is expected
+when making requests inside opaque input request objects and unspecified output
+responses, that mechanism is JSON Schema Object definitions. This allows Broker
+authors to programmatically define inputs and outputs for applicable requests.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| parameters | JSON schema object | The schema definition for the input parameters. Each input parameter is expressed as a property within a JSON object. |
+| parameters | JSON schema object | The schema definition for the input `parameters`. Each input parameter is expressed as a property within a JSON object. |
+| response | JSON schema object | The schema definition for the output response. |
+
+The following rules apply if `parameters` or `response` included anywhere in the catalog:
```

Comment:
"The following rules apply if parameters or response **IS** included"...

Reply:
Thanks! Using "are". Ex: If thing1 or thing2 are included. vs If thing1 or thing2 is included.